packages/salesforcedx-vscode-apex/src/commands/forceApexTestRun.ts

```diff
@@ -162,8 +163,12 @@ function getTempFolder(): string {
 export class ForceApexTestRunExecutor extends SfdxCommandletExecutor<
   ApexTestQuickPickItem
 > {
+  constructor() {
+    super(OUTPUT_CHANNEL);
+  }
+
```

Comment:
We should introduce the output channel changes for the `forceApexTestRunCodeAction.ts` file as well. If you run tests through the side bar or the code lens within the files, the output is displayed in the `Salesforce CLI` channel. Running the tests through the command palette displays the output in the `Apex Extension` channel - same as the other Apex commands. We should keep this consistent so users aren't confused on where to look for their test results